docs/gallery/interactive/pyfar_orientations.ipynb

```diff
@@ -0,0 +1,133 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Orientations\n",
```

Comment:
This title will show up on the thumbnail in the gallery. Maybe be more illustrative with something like 'Working with pyfar Orientations' or 'Handling Orientations'?